easycv/datasets/detection/raw.py

```diff
@@ -107,22 +108,24 @@ def visualize(self, results, vis_num=10, score_thr=0.3, **kwargs):
         elif hasattr(self.data_source, 'classes'):
             class_names = self.data_source.classes
 
+        # If class_names is not None, class_id will be converted to class_name for visualization,
+        # otherwise the class_id will be displayed.
```

Comment:
I mean when should user provide class_names and when not, detection_classes has different meaning  when class_names exists

Reply:
It’s a auto conversion and only be done during visualization. In fact, user does not need to care, and detection_classes here is just a temporary variable

Reply:
this point should be made clear for our self-maintainance